frontend/src/locales/scripts/en.json5

```
@@ -80,14 +80,14 @@
     },
     collection: {
       content: {
-        a: "{openverse} searches across more than 700 million images and audio from open APIs and the {commonCrawl} dataset.",
+        a: "{openverse} searches across more than 700 million images and audio track from open APIs and the {commonCrawl} dataset.",
```

We would use the plural form of the word here as we are referring to more than one audio track in this string.

```suggestion
        a: "{openverse} searches across more than 700 million images and audio tracks from open APIs and the {commonCrawl} dataset.",
```